src/lib.rs

```diff
@@ -156,6 +159,13 @@ compile_error!("A least one underlying storage type must be enabled. See the fea
 #[doc(hidden)]
 pub extern crate num;
 
+#[doc(hidden)]
```

Comment:
Looks like `cfg_attr` isn't necessary. The compiler is smart enough that `#[doc(hidden)]` doesn't "fall through" to the next item when the `serde` feature is not enabled.